src/components/CheckForm/CheckForm.tsx

```diff
@@ -123,6 +124,70 @@ const CheckFormContent = ({ check, checkType, overCheckLimit, overScriptedLimit
     deleteCheck(check, { onSuccess });
   };
 
+  const actions = useMemo(() => {
+    const actions = [
```

Comment:
This is probably the code smelliest thing. I'm kind of unsure about delegating the responsibility for rendering actions to FormLayout. It's not super ergonomic and requires some prop drilling. It was the quickest way I could see to get the desired layout, though